### DISCUSSION TITLE
v0.0.17 SOCKS5 UDP Client support

### DISCUSSION THREAD

**Discussion by fortuna** [OutlineFoundation/outline-sdk#288] [Announcements]
We're excited to announce that our [SOCKS5 client library](https://pkg.go.dev/github.com/Jigsaw-Code/outline-sdk/transport/socks5) now supports UDP! You can now utilize SOCKS5 proxies to measure the performance of DNS, QUIC, and other UDP-based protocols, which was only possible over Shadowsocks before. This will open up the possibilities for remote testing.

This improvement (https://github.com/Jigsaw-Code/outline-sdk/pull/257) was made possible thanks to the valuable contributions of @amircybersec as part of his OTF ICFP Fellowship.

**Full Changelog**: https://github.com/Jigsaw-Code/outline-sdk/compare/v0.0.16...v0.0.17

<hr /><em>This discussion was created from the release <a href='https://github.com/Jigsaw-Code/outline-sdk/releases/tag/v0.0.17'>v0.0.17 SOCKS5 UDP Client support</a>.</em>

**Comment by daokdo123** [comment]
不支持下载吗？
